public/app/plugins/datasource/grafana-azure-monitor-datasource/types/query.ts

```diff
@@ -47,6 +47,7 @@ export interface AzureMetricQuery {
   resourceName?: string;
   /** Resource type */
   metricNamespace?: string;
+  customNamespace?: string;
```

Comment:
since we are differing from the naming in Azure Monitor API, it would be good to add some comment here explaining the difference between `metricNamespace` and `customNamespace` for future reference

Reply:
Yeah, I can PR that this am.